Changes have been made to home.css and search.css files in the artadosearch/css directory.

OpenSoftware-World opened the pull request:
Changes have been made to home.css and search.css files in the artadosearch/css directory.

OpenSoftware-World committed:
artadosearch/css dizinindeki home.css ve search.css dosyalarında değişiklik yapılmıştır.

ardatdev commented:
Tema için ekipce bir görüşme yapılmalı. Şu anda ekibe alımlar yapmaktayız. Discord sunucusuna gelerek katılabilirsin ekibe.

ardatdev committed:
Update home.css

Değişiklikleri kod içinde yorum satırı olarak değil. Commit'de açıklama olarak yaz. Ayrıca kendi imzanı atmana gerek yok. Zaten commit'i senin yaptığın gözüküyor.

ardatdev committed:
Update search.css

ardatdev commented:
Commit açıklamasında da dediğim gibi değişiklikleri kod içinde yorum satırı olarak değil. Commit'de açıklama olarak yaz. Ayrıca kendi imzanı atmana gerek yok. Zaten commit'i senin yaptığın gözüküyor. Ayrıca lütfen pull request ve issue'larda İngilizce kullanalım.

LinuxUsersLinuxMint commented:
> Commit açıklamasında da dediğim gibi değişiklikleri kod içinde yorum satırı olarak değil. Commit'de açıklama olarak yaz. Ayrıca kendi imzanı atmana gerek yok. Zaten commit'i senin yaptığın gözüküyor. Ayrıca lütfen pull request ve issue'larda İngilizce kullanalım.

tamamdır özür dilerim Bundan sonra ingilizce kullanmaya dikkat edeceğim :)

LinuxUsersLinuxMint commented:
> scord sunucusuna gelerek katılabilirsin ekibe

discord sunucunuzun adı nedir?

ardatdev commented:
Artado Search'ın ana sayfasında, menü kısmında var davet linki.
